packages/playwright/src/generators/init/utils/update-targets.ts

```diff
@@ -39,21 +44,19 @@ export function updateProjectJsonWithNativeVisualRegressionTargets(
                     options: {
                         commands: [
                             {
-                                command:
-                                    'docker run -v $PWD:/project -w /project --rm --ipc=host mcr.microsoft.com/playwright:v1.30.0-jammy npx nx run next-custom-server-e2e:e2e',
+                                command: `docker run -v $PWD:/project -w /project --rm --ipc=host mcr.microsoft.com/playwright:${playwrightVersion}-jammy npx nx run ${project.name}:e2e`,
                                 forwardAllArgs: false,
                             },
                         ],
                     },
                     configurations: {
                         updatesnapshots: {
-                            args: '--extra=--update-snapshots',
+                            args: '--extra="--update-snapshots --grep @visual-regression"',
                         },
                         compilearm64: {
                             commands: [
                                 {
-                                    command:
-                                        'docker run -v $PWD:/project -w /project --rm --ipc=host mcr.microsoft.com/playwright:v1.30.0-jammy /bin/bash -c "apt update && apt install -y make gcc g++ && rm -rf node_modules/@parcel && npm install" && npm install',
+                                    command: `docker run -v $PWD:/project -w /project --rm --ipc=host mcr.microsoft.com/playwright:${playwrightVersion}-jammy /bin/bash -c "apt update && apt install -y make gcc g++ && rm -rf node_modules/@parcel && npm install" && npm install`,
```

Comment:
Yikes. Hopefully we don't need this forever

Reply:
Hopefully not, but with current arm64 architecture it needs to be done as a 1 time activity

Reply:
Why two `npm install`s? Is the `&&` cross platform?

Reply:
@johnmckenna

Reply:
You need to install it inside the docker image and install again locally otherwise things start to fail locally